### PR TITLE
Feature spf2 25

### PR DESCRIPTION
**Pull request by Russ07**
Get all inventory items

### CODE REVIEW

File: controllers/inventoriesControllers.js [hunk 1]
```diff
@@ -84,6 +84,19 @@ const createNewInventory = async (req, res) => {
...
+    const allInventoryItems = await knex("inventories");
+    if (allInventoryItems.length === 0) {
+      return res.status(200).json({ message: 'Currently, inventory is empty' });
+    }
+    res.status(200).json(allInventoryItems);
+  } catch (error) {
+    res.status(500).json({ error: "Failed to fetch all inventory items" });
+  }
+};
+
```
Comment (bernard493) [inline controllers/inventoriesControllers.js:98]: If you check your ticket it say  **Note**: The warehouse name for each inventory item must also be included in the response. 
 i dont think you doing that

File: controllers/inventoriesControllers.js [hunk 2]
```diff
@@ -84,6 +84,19 @@ const createNewInventory = async (req, res) => {
   }
 };
 
+const getAllInventoryItems = async (req, res) => {
+  try {
+    const allInventoryItems = await knex("inventories");
+    if (allInventoryItems.length === 0) {
+      return res.status(200).json({ message: 'Currently, inventory is empty' });
```
Comment (bernard493) [inline controllers/inventoriesControllers.js:91]: If you check your ticket it say  **Note**: The warehouse name for each inventory item must also be included in the response. 
 i dont think you doing that

File: controllers/inventoriesControllers.js [hunk 3]
```diff
@@ -84,6 +84,19 @@ const createNewInventory = async (req, res) => {
   }
 };
 
+const getAllInventoryItems = async (req, res) => {
+  try {
+    const allInventoryItems = await knex("inventories");
```
Comment (Diego-Romero) [inline controllers/inventoriesControllers.js]: Nit: we could consider adding these string such as "inventories" to a constants file, that way we don't have to write it every time, and if we need to refactor, we can simply do it from that file.

File: controllers/inventoriesControllers.js (hunk 2 above)
Comment (Diego-Romero) [reply controllers/inventoriesControllers.js:91]: What do you mean by this @bernard493 could you provide an example? In this line of code there are no inventory items, should an empty array be provided?

Comment (bernard493) [reply controllers/inventoriesControllers.js:91]: @Diego-Romero 
```
from this code the response of  an inventory will be 
{
  "warehouse_id": 1,
  "item_name": "Paper Towels",
  "description": "Made out of military-grade synthetic materials, these paper towels are highly flammable, yet water resistant, and easy to clean.",
  "category": "Gear",
  "status": "Out of Stock",
  "quantity": 0
}

but the ticket says  we need to add the warehouse name to the inventories res

 "warehouse_name": "Manhattan",

   {
      "id": 1,
      "warehouse_name": "Manhattan",
      "item_name": "Television",
      "description": "This 50\", 4K LED TV provides a crystal-clear picture and vivid colors.",
      "category": "Electronics",
      "status": "In Stock",
      "quantity": 500
    },
```

File: controllers/inventoriesControllers.js [hunk 4]
```diff
@@ -72,6 +72,19 @@ const createNewInventory = async (req, res) => {
   }
 };
 
+
+const getAllInventoryItems = async (req, res) => {
+  try {
+    const allInventoryItems = await knex("inventories");
+    if (allInventoryItems.length === 0) {
+      return res.status(200).json({ message: 'Currently, inventory is empty' });
```
Comment (Diego-Romero) [inline controllers/inventoriesControllers.js:91]: This should be an empty array, as having no inventory is a valid state.

**Pull request merged**